tests/test_machine_only_rules.py

```diff
@@ -21,58 +31,94 @@ def main():
         guide_dir = os.path.abspath(
             os.path.join(product_dir, product_yaml['benchmark_root']))
         additional_content_directories = product_yaml.get("additional_content_directories", [])
-        add_content_dirs = [os.path.abspath(os.path.join(product_dir, rd)) for rd in additional_content_directories]
+        add_content_dirs = [os.path.abspath(
+            os.path.join(product_dir, rd)) for rd in additional_content_directories]
         if not check_product(args.build_dir, product, [guide_dir] + add_content_dirs):
             test_result = 1
     if test_result:
         sys.exit(1)
 
+
 def check_product(build_dir, product, rules_dirs):
     input_groups, input_rules = scan_rules_groups(rules_dirs, False)
     ds_path = os.path.join(build_dir, "ssg-" + product + "-ds.xml")
-    if not check_ds(ds_path, "Group", input_groups):
-        return False
-    if not check_ds(ds_path, "Rule", input_rules):
+    if machine_platform_missing_in_rules(ds_path, input_rules):
```

Comment:
The input_groups is unused now unused. Is that intentional?

Reply:
Yes, this is intentional. We currently use groups only when reading the directory structure (rule.yamls, group.yamls) and determining if the rule is inheriting the "machine" platform. But we are not checking their platform attribute in XCCDF anymore.

Reply:
so you can just replace `input_groups` by `_`

Reply:
Correct, fixed. Thanks.